style.css

```diff
@@ -0,0 +1,189 @@
+:root {
+  --dark-grey-blue: #333d4b;
+  --dark-cyan: #0e8784;
+  --pale-orange: #fdd6ba;
+  --light-cream: #fefcf7;
+  --grey: #83888f;
+  --white: #ffffff;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
```

Comment:
Add container for whole page, which will have max-width (for example 1280px), because: 
![image](https://github.com/ksawerykasp/coffeeroasters/assets/76774237/e6f833b5-579c-49ab-8f55-f6e86f09e427)